ResearchKit/Common/ORKPasscodeViewController.h

```diff
@@ -66,6 +66,22 @@ ORK_AVAILABLE_DECL
  */
 - (void)passcodeViewControllerDidCancel:(UIViewController *)viewController;
 
+/*
```

Comment:
I do not, but if you look at RK or UITableViewDelegate or UITableViewDataSource... or anything else where Apple is using the delegate pattern, they include "who sent this". Since this is going into RK, I'd like to keep consistent with the `- passcodeViewControllerDidCancel:` method defined above this one. ;)